packages/components/organisms/f-checkout/src/services/tests/addressService.test.js

```diff
@@ -174,4 +175,90 @@ describe('addressService', () => {
             });
         });
     });
+
+    describe('isAddressInLocalStorage ::', () => {
```

Comment:
Could you add arrange act to these please?

Reply:
Done